knowledge-base/kb-radwindow-prismapplication-createshell.md

```diff
@@ -0,0 +1,52 @@
+---
+title: CreateShell() method can't return RadWindow in Prism 7
```

Comment:
Also, please use capital letters for the words of the title as in most of the other articles.